app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,35 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: Rails.configuration.folio_sync['default_sender_email_address']
+  layout 'mailer'
+
+  def folio_sync_error_email
+    body_content = "One or more errors were encountered during FOLIO sync:\n\n"
```

Comment:
There is a chance that there could be a LOT of errors in one sync. Would it be possible to add total number of errors (separated by type) up top? Do we need to worry about this email being too large, and what to do if that's the case (I think this might be a question for @elohanlon )?

Reply:
Done! Added number of errors (total, downloading errors and syncing errors) + now we list up to 50 errors each in the email.